packages/unminify/src/transformations/un-es6-class.ts

```diff
@@ -1,8 +1,14 @@
+import { findReferences } from '@wakaru/ast-utils'
+import { findHelperLocals, removeHelperImport } from '../utils/import'
 import wrap from '../wrapAstTransformation'
 import type { ASTTransformation } from '../wrapAstTransformation'
 import type { ExpressionKind } from 'ast-types/lib/gen/kinds'
+import type { Scope } from 'ast-types/lib/scope'
 import type { AssignmentExpression, CallExpression, ExpressionStatement, FunctionExpression, Identifier, MemberExpression, VariableDeclarator } from 'jscodeshift'
 
+const inheritsModuleName = '@babel/runtime/helpers/inherits'
+const inheritsModuleEsmName = '@babel/runtime/helpers/esm/inherits'
```

Comment:
These will only work in the case of Babel right? What about when `swc` or otherwise is used?

Also, does it make sense to define these paths here within `packages/unminify/src/transformations/un-es6-class.ts`? They feel like they're more generic and might belong in a more 'shared babel space'?

Reply:
Yes, it only works for Babel. Currently only Babel is supported. SWC and TypeScript should be supported in the future.